L1Trigger/TrackFindingTracklet/src/TrackletConfigBuilder.cc

```diff
@@ -1130,110 +1130,110 @@ void TrackletConfigBuilder::writeILMemories(std::ostream& os, std::ostream& memo
   layerdisk[15] = 9;
   phimin[15] = -0.0963091;
   phimax[15] = 0.794441;
-  dtcname[16] = "2S_1";
-  layerdisk[16] = 3;
-  phimin[16] = -0.0246209;
-  phimax[16] = 0.763311;
-  dtcname[17] = "2S_1";
-  layerdisk[17] = 4;
-  phimin[17] = 0.261875;
-  phimax[17] = 0.403311;
-  dtcname[18] = "2S_2";
-  layerdisk[18] = 4;
-  phimin[18] = -0.0542445;
-  phimax[18] = 0.715509;
-  dtcname[19] = "2S_3";
-  layerdisk[19] = 5;
-  phimin[19] = 0.0410126;
-  phimax[19] = 0.730605;
-  dtcname[20] = "2S_4";
-  layerdisk[20] = 5;
-  phimin[20] = -0.0428961;
-  phimax[20] = 0.693862;
-  dtcname[21] = "2S_4";
-  layerdisk[21] = 8;
-  phimin[21] = -0.0676705;
-  phimax[21] = 0.765802;
-  dtcname[22] = "2S_5";
-  layerdisk[22] = 6;
-  phimin[22] = -0.0648206;
-  phimax[22] = 0.762952;
-  dtcname[23] = "2S_5";
-  layerdisk[23] = 9;
-  phimin[23] = -0.0676705;
-  phimax[23] = 0.765802;
-  dtcname[24] = "2S_6";
+  dtcname[16] = "negPS10G_1";
```

Comment:
I expect all these magic numbers to give us problems with PR to official CMSSW.
In my Email to Anders/Emyr of 24th Sept 2020, I supplied code to calculate the phi range of each DTC. e.g. https://github.com/tomalin/cmssw/blob/ian_TxtFiles/TrackFindingTrackletHLS/src/TxtFileWriter.cc#L30 . So is the only reason we are hard-wiring these numbers now because of the desire to run TrackletConfigBuilder outside CMSSW?